AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_OcclusionCullingPlaneAdded.py

```diff
@@ -0,0 +1,159 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    creation_undo = (
+        "UNDO Entity creation success",
+        "UNDO Entity creation failed")
+    creation_redo = (
+        "REDO Entity creation success",
+        "REDO Entity creation failed")
+    occlusion_culling_plane_entity_creation = (
+        "Occlusion Culling Plane Entity successfully created",
+        "Occlusion Culling Plane Entity failed to be created")
+    occlusion_culling_plane_component_added = (
+        "Entity has a Occlusion Culling Plane component",
+        "Entity failed to find Occlusion Culling Plane component")
+    enter_game_mode = (
+        "Entered game mode",
+        "Failed to enter game mode")
+    exit_game_mode = (
+        "Exited game mode",
+        "Couldn't exit game mode")
+    is_visible = (
+        "Entity is visible",
+        "Entity was not visible")
+    is_hidden = (
+        "Entity is hidden",
+        "Entity was not hidden")
+    entity_deleted = (
+        "Entity deleted",
+        "Entity was not deleted")
+    deletion_undo = (
+        "UNDO deletion success",
+        "UNDO deletion failed")
+    deletion_redo = (
+        "REDO deletion success",
+        "REDO deletion failed")
+
+
+def AtomEditorComponents_occlusion_culling_plane_AddedToEntity():
```

Comment:
```suggestion
def AtomEditorComponents_OcclusionCullingPlane_AddedToEntity():
```
this is the way that was established for naming these. it is what it is.

Reply:
Fixed!

Reply:
For context, we name it this way because we use the Osherove naming convention - more reading can be found here: https://osherove.com/blog/2005/4/3/naming-standards-for-unit-tests.html 

In short, the format is `UnitOfWork_StateUnderTest_ExpectedBehavior` which is why we format it that way.